src/lib/duration/create.js

```diff
@@ -38,7 +38,7 @@ export function createDuration (input, key) {
             duration.milliseconds = input;
         }
     } else if (!!(match = aspNetRegex.exec(input))) {
-        sign = (match[1] === '-') ? -1 : 1;
+        sign = (match[1] === '-') ? -1 : (match[1] === '+') ? 1 : 1;
```

Comment:
It's unusual to see `condition ? foo : foo` as it simplifies to just `foo`.

Put another way: this change has no effect.  The original line of code works just as well.